### ISSUE TITLE
Hardcoded Django add new popup window not working if nested

### ISSUE THREAD

**Issue by kalevhark** [crucialfelix/django-ajax-selects#310]
Tested new v3.0.1 in Django admin but it's don't work correctly in the case of nested popups.

Steps I did: 
1. in ajax-select-multiple widget selected "add new" item
2. opened new item popup window
3. in new popup window select "add new" item, I fiiled data and saved
4. popup 2 closes but in popup 1 nothing happens.

I propose problem is ajax-select.js code in line 244:
var input = $("#" + win.name.replace("__1", ""));
I changed to:
var input = $("#" + win.name.split("__")[0]);
And in my case it's now works.

Problem seems to be in enumerating of popups - every popup has unique number in the end "__1", "__2" etc.

**Comment by crucialfelix** [comment]
Yes, applying hacks like this is always going to run into problems.  Django isn't intending that plugins hack their system to extend functionality.

I will test it with removing all the __N, but I think it's going to cause more problems.

Thanks very much for reporting!
